prefab_cloud_python/client.py

```diff
@@ -110,5 +123,9 @@ def post(self, path, body):
             "Authorization": f"Basic {auth_token}",
         }
 
-        endpoint = self.options.prefab_api_url.strip("/") + "/" + path.strip("/")
+        endpoint = (
+            self.options.prefab_api_url.strip("/") + "/" + path.strip("/")
+            if self.options.prefab_api_url
+            else ""
+        )
```

Comment:
nit: can we use [urlparse.parse.join()](https://docs.python.org/3.11/library/urllib.parse.html#urllib.parse.urljoin) for URL manipulation stuff? (separate PR is fine)